collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -190,7 +190,12 @@ private void processBeanValue(
             Object value) {
         if (value == null) {
             logScrape(domain + beanProperties + attrName, "null");
-        } else if (value instanceof Number || value instanceof String || value instanceof Boolean) {
+        } else if (value instanceof Number || value instanceof String || value instanceof Boolean || value instanceof java.util.Date) {
+            //output Date as Long (the number of milliseconds since January 1, 1970, 00:00:00 GMT)
```

Comment:
Seconds are usual for Prometheus.

Reply:
updated to output the number of seconds instead of milliseconds.